doc/api/net.md

```diff
@@ -281,6 +281,14 @@ added: v0.1.90
 
 Emitted when the server has been bound after calling [`server.listen()`][].
 
+### Event: `'drop'`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Emitted when the server drops a connection.
```

Comment:
Could explain in which condition a connection is dropped?

Reply:
> Could explain in which condition a connection is dropped?

Thanks ! Done !